src/lib/moment/get-set.js

```diff
@@ -27,11 +29,27 @@ export function set (mom, unit, value) {
 
 // MOMENTS
 
-export function getSet (units, value) {
-    var unit;
+export function stringGet (units) {
+    units = normalizeUnits(units);
+    if (isFunction(this[units])) {
+        return this[units]();
+    }
+    return this;
+}
+
+
+export function stringSet (units, value) {
     if (typeof units === 'object') {
-        for (unit in units) {
-            this.set(unit, units[unit]);
+        units = normalizeObjectUnits(units);
+        // for (unit in units) {
+        //     this.set(unit, units[unit]);
+        // }
+        var prioritized = getPrioritizedUnits();
+        for (var i = 0; i < prioritized.length; i++)
```

Comment:
The identity lookup makes sense. When I wrote this code I was thinking 'how many unit types could possibly be added', but I suppose if calendar systems were added, a lot.